src/commons/BrowserUtils.ts

```diff
@@ -247,16 +247,39 @@ export namespace BrowserUtils {
     return $(selector).isEnabled();
   }
 
+  /**
+   *  Wait for element to be enabled
+   * @param selector element selector
+   */
+  export function waitForEnabled(selector: string): void {
+    Reporter.debug(`Wait for an element to be enabled '${selector}'`);
+    isExist(selector);
```

Comment:
isExist actually waitForExist We need rename isExist to waitForExist and add isExist method

Reply:
done

Reply:
proven to be stable

Reply:
changed for waitForDisplayed

Reply:
changed to waitForDisplayed